docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/templates/content/blog/node--cgov-blog-post--lb-primary-content.html.twig

```diff
@@ -30,7 +30,7 @@
     {{ drupal_block('cgov_blog_pager', {label: ''}) }}
     {% if content.field_recommended_content[0] %}
       <div id="blog-cards">
-        <h4>{{ series.field_recommended_content_header.value }}</h4>
+        <h2 class="related-resources__header">{{ series.field_recommended_content_header.value }}</h2>
```

Comment:
the class name should include recommended-content - not related-resources, its a different section